Get snapshots support for multiple repositories (#42090)
elastic/elasticsearch: docs/reference/modules/snapshots.asciidoc

such as status information, the number of snapshotted shards, etc.  The default
value of the `verbose` parameter is `true`.

> It is also possible to retrieve snapshots for multiple repositories in one go, for example:

Reviewer: ```suggestion
It is also possible to retrieve snapshots from multiple repositories at once, for example:
```
